app/Sources/TuistApp/AuthenticationService.swift

```diff
@@ -0,0 +1,53 @@
+#if os(iOS)
```

Comment:
I'm keeping refactoring the authentication in the macOS app for a follow-up, so this is currently iOS only, but only temporarily so.